lib/src/middleware/routing_middleware.dart

```diff
@@ -0,0 +1,57 @@
+import '../adapter/context.dart';
+import '../handler/handler.dart';
+import '../method/request_method.dart';
+import '../router/router.dart';
+import 'middleware.dart';
+
+Middleware routeWith(final Router<Handler> router) =>
+    RoutingMiddleware(router).meddle;
+
+final _pathParametersStorage = Expando<Map<Symbol, String>>();
+
+class RoutingMiddleware {
+  final Router<Handler> _router;
+
+  RoutingMiddleware(this._router);
+
+  Handler meddle(final Handler next) {
```

Comment:
Should I know what `meddle` means in this context?

As a side-note, probably this project should also pull in lints like the Serverpod ones to document the public methods, right?

Reply:
I have a `<X>Middleware` class that should produce a `Middleware` (`typedef Middleware = Handler Function(Handler)`). What should I call this function? I went with `meddle` since it is in the middle meddling with stuff. I'm open to suggestions 😄